ui-grid/ui-grid.d.ts

```diff
@@ -2871,13 +2871,13 @@ declare module uiGrid {
              * @param {any} rowEntity gridOptions.data[] array value
              * @param {ng.IAngularEvent} event object if raised from event
              */
-            toggleRowSelection(rowEntity: IGridRow, event?: ng.IAngularEvent): void;
+            toggleRowSelection(rowEntity: uiGrid.IGridRow, event?: ng.IAngularEvent): void;
```

Comment:
Same here. rowEntity is an object in the gridOptions.data array, not uiGrid.IGridRow.